crates/runtime_isolate/Cargo.toml

```diff
@@ -15,10 +15,21 @@ log = { version = "0.4.18", features = ["std", "kv_unstable"] }
 once_cell = "1.17.1"
 async-recursion = "1.0.4"
 linked-hash-map = "0.5.6"
+flate2 = "1.0.26"
+serde_v8 = "0.100.0"
+serde = { version = "1", features = ["derive"] }
+serde_json = "1"
+uuid = { version = "1.3.3", features = ["v4"] }
 lagon-runtime-v8-utils = { path = "../runtime_v8_utils" }
 lagon-runtime-http = { path = "../runtime_http" }
 lagon-runtime-crypto = { path = "../runtime_crypto" }
 
 [features]
 default = []
 ignore-snapshot = []
+
+[profile.bench.package.flate2]
+opt-level = 3
+
+[profile.release.package.flate2]
+opt-level = 3
```

Comment:
I think this should be moved to the root `Cargo.toml`?

Reply:
done